server/crawler/index.ts

```diff
@@ -257,3 +257,20 @@ export default async function getCralwerData(url: string) {
   }
   return { title, description, keywords, author, body, ogImage, images }
 }
+
+export async function crawlOgImage(url: string) {
+  const { data } = await axios.get(encodeURI(url as string))
+  const $ = cheerio.load(data)
+  const metas = $('meta')
+  let ogImageUrl = ''
+  for (let i = 0; i < Object.keys(metas).length; i += 1) {
+    const { attribs } = metas[i]; // keys are integers 1 to N
+    if (attribs?.property === 'og:image') {
+      ogImageUrl = attribs.content
+      break
+    }
+  }
+  if (!ogImageUrl) return null;
```

Comment:
Just reuse `crawlData()` here?

Reply:
Sure we can get all the necessary info in `crawlData()`, including ogImage. But because we don't need to cache ogImage URL when registering ISCN. I think we can actually remove searching ogImage logic in `crawlData()`. Also, `crawlData()` calls additional async functions like `Promise.all(promiseImg)` that creating class ID doesn't need. Is it preferred to separate the crawling logic into two functions?

Reply:
Hmm yes the function got complex after arweave image support... lets just use this version here then